cmd/serverless/main.go

```diff
@@ -203,7 +203,7 @@ func runAgent(stopCh chan struct{}) (serverlessDaemon *daemon.Daemon, err error)
 	config.Datadog.SetConfigFile(datadogConfigPath)
 	// Load datadog.yaml file into the config, so that metricAgent can pick these configurations
 	if _, err := config.Load(); err != nil {
-		log.Errorf("Error happened when loading configuration from datadog.yaml for metric agent: %w", err)
+		log.Errorf("Error happened when loading configuration from datadog.yaml for metric agent: %s", err)
```

Comment:
Oh interesting! I am assuming it did not like the `%w`? I guess that makes sense. Sorry for misleading you.

Reply:
Yes. It does not like %w. I guess it is due to the logger here is our own wrapped one.